include/ofi_hmem.h

```diff
@@ -43,6 +43,18 @@
 
 extern bool ofi_hmem_disable_p2p;
 
+#define IPC_HANDLE_SIZE		64
+
+struct ipc_info {
```

Comment:
why there are `ipc_info` and `ofi_mr_hmem_ipc_info`?

Reply:
The `ipc_info` is used by shm cmd queue. `ofi_mr_hmem_ipc_info` is the minimum ipc info added to ofi_mr_entry->info.

Reply:
What is the relationship between them? Can we store `ipc_info` in `ofi_mr_entry` too?

Reply:
It seems that `ipc_info` was `shm_ipc_info`. The reason it is moved to `ofi_hmem.h` is because `ofi_ipc_cache_search` now takes it as input, and use it to construct a `ofi_mr_info`.

I think it is better to remain `ipc_info` as `shm_ipc_info` and keep it inside shm. shm provider prepare `ofi_mr_info` and call `ofi_ipc_cache_search` with it.

Reply:
Move `ipc_info` to `smr_ipc_info` in `ofi_shm.h` in the latest revision.

Reply:
continuous-integration/appveyor/pr — AppVeyor build failed because ofi_shm.h cannot be compiled on windows. @wzamazon I think it is because I moved ipc_info to ofi_shm.h and I had to include ofi_shm.h in hmem_ipc_cache.c. Maybe I should move ipc_info (and IPC_HANDLE_SIZE) back to ofi_hmem.h